pkg/resource/watcher.go

```diff
@@ -36,6 +36,12 @@ func GetPodWatcher(resourceType, resourceName string) PodWatcher {
 
 			matchLabels = service.Spec.Selector
 
+		case "pods", "po":
```

Comment:
```suggestion
		case "po", "pod", "pods:
```

We aim to support all valid possibilities